docs/network-flow-visibility.md

```diff
@@ -764,6 +764,12 @@ the selected time range.
 
 <img src="https://downloads.antrea.io/static/04132023/flow-visibility-network-topology-0.png" width="400" alt="Network Topology Dashboard service dependency graph">
 
+Users can group Pods by label, allowing Pod squares in the diagram to represent
```

Comment:
maybe add some instructions to let users know where should they find the configurations, telling users it is in panel edit